index.html

```diff
@@ -11,69 +11,339 @@
 </head>
 
 <body>
-    <header>
+    <section class="toolbar">
+        <div class="nav-menu">
+            <a href="#">Welcome</a>
+            <div class="link-list">
+                <a href="#">project</a>
+                <a href="#">About</a>
+                <a href="#">Information</a>
+            </div>
+            <a href="#" class="menu-icon">
+                <i class="fa fa-bars"></i></a>
+        </div>
 
-        <section class="toolbar">
-            <div class="nav-menu">
-                <a href="#">Welcome</a>
-                <div class="link-list">
-                    <a href="#">project</a>
-                    <a href="#">About</a>
-                    <a href="#">Information</a>
-                </div>
-                <a href="#" class="menu-icon">
-                    <i class="fa fa-bars"></i></a>
+        <section class="headline">
+            <div class="primary-text">
+                <h1>Hey there, I'm Addis</h1>
+                <h4>I'm software Engineer</h4>
+            </div>
+            <div class="supporting-text">
+                <p>I develop websites and other software products with high quality.
+                    I have a series of experiences in building software for different customers. If you have any
+                    project, please contact me.
+                </p>
+                <p>below my contact</p>
+            </div>
+            <div class="social-media">
+                <ul class="social-nav">
+                    <li><a href="http://linkedin.com/AddisTsega"><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href="http://linkedin.com/AdaTsega"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="http://linkedin.com/Aseadd"><i class="fa fa-github"></i></a></li>
+                    <li><a href="http://linkedin.com/Addis-Tsega"><i class="fa fa-facebook"></i></a></li>
+                </ul>
```

Comment:
Nice UI @Aseadd. 🤝

- [ ] [Optional] It is better if you make the project similar to what is provided in the Figma design. Kindly change the color of your social media icons.
- Your design
![Yours](https://user-images.githubusercontent.com/63917725/177014248-8ea58201-f239-417b-8f73-25a05f507b13.PNG)